modules/xmpp/strophe.jingle.js

```diff
@@ -149,14 +149,20 @@ export default class JingleConnectionPlugin extends ConnectionPlugin {
             logger.info(
                 `Marking session from ${fromJid
                 } as ${isP2P ? '' : '*not*'} P2P`);
+
+            const iceConfig = isP2P ? this.p2pIceConfig : this.jvbIceConfig;
+
             sess
                 = new JingleSessionPC(
                     $(iq).find('jingle').attr('sid'),
                     $(iq).attr('to'),
                     fromJid,
                     this.connection,
                     this.mediaConstraints,
-                    isP2P ? this.p2pIceConfig : this.jvbIceConfig,
+
+                    // Makes a copy in order to prevent exception thrown on RN when either this.p2pIceConfig or
+                    // this.jvbIceConfig is modified and there's a PeerConnection instance holding a reference
+                    { ...iceConfig },
```

Comment:
This only does a shallow clone. In order to be extra safe I'd do `JSON.parse(JSON.stringify(data));`